axon-server-connector/src/test/java/org/axonframework/axonserver/connector/query/StreamingQueryEndToEndTest.java

```diff
@@ -48,16 +49,21 @@
 import java.time.Duration;
 import java.util.List;
 import java.util.Objects;
+import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicReference;
 
 import static java.util.Arrays.asList;
+import static org.axonframework.axonserver.connector.util.ExecutorServiceBuilder.THREAD_KEEP_ALIVE_TIME;
 import static org.axonframework.messaging.responsetypes.ResponseTypes.multipleInstancesOf;
 import static org.junit.jupiter.api.Assertions.*;
 
 /**
  * End-to-end tests for Streaming Query functionality. They include backwards compatibility end-to-end tests as well.
  */
-@Disabled("Disabled until Axon Server 4.6 is released.")
+//@Disabled("Disabled until Axon Server 4.6 is released.")
```

Comment:
This should just be removed right? I guess this was just to quickly test of it ran successfully now?